### DISCUSSION TITLE
Nach Installation via SSH fehlt CSS

### DISCUSSION THREAD

**Discussion by Rayankhalo7** [kimai/kimai#4645] [Questions & Answers]
Ich habe Kimai via SSH auf meine Domain installiert. Nach der Installation fehlt CSS. Kann mir eventuell helfen woran das liegen kann?![image](https://github.com/kimai/kimai/assets/160142002/bb0a0f02-35b0-4882-8de7-9cb3134f9896) ![image](https://github.com/kimai/kimai/assets/160142002/2f7c6231-6a97-45bd-8b9b-418e149ecdbc)

**Comment by kevinpapst** [comment]
This was already answered multiple times. Looks like a installation in a subdirectory, which is not possible. Point your domain to public/ as described in the docs

**Comment by Rayankhalo7** [comment]
Amazing. Thank you  very much.
